test/e2e_node/util.go

```diff
@@ -91,9 +91,29 @@ var (
 	kubeletHealthCheckURL    = fmt.Sprintf("http://127.0.0.1:%d/healthz", ports.KubeletHealthzPort)
 	containerRuntimeUnitName = ""
 	// KubeletConfig is the kubelet configuration the test is running against.
-	kubeletCfg *kubeletconfig.KubeletConfiguration
+	kubeletCfg  *kubeletconfig.KubeletConfiguration
+	featureGate featuregate.FeatureGate
```

Comment:
Is this variable really needed? Can't the `utilfeature.DefaultMutableFeatureGate` be used instead?

Reply:
Hey @pohly! Thanks for reviewing!

That's a good question.
The previous function from `skipper` package was cloning `utilfeature.DefaultMutableFeatureGate`.

Do you think there's a good reason to avoid mutating `utilfeature.DefaultMutableFeatureGate` directly? If not I'd be happy to get rid of the `featureGate` variable.

Reply:
A package should not mutate utilfeature.DefaultMutableFeatureGate, but `test/e2e_node` is the "main binary" and can modify it.

There is also an `AddFlag` method, but I think that needs pflag and thus can't be used here.